src/app/public/modules/dynamic-component/dynamic-component.service.ts

```diff
@@ -85,4 +85,20 @@ export class SkyDynamicComponentService {
     return cmpRef;
   }
 
+  /**
+   * Removes a component ref from the page
+   * @param cmpRef Component ref for the component being removed
+   */
+  public removeComponent<T>(
+    cmpRef: ComponentRef<T>
+  ): void {
+    const bodyEl = this.windowRef.getWindow().document.body;
+
+    // Technique for retrieving the component's root node taken from here:
+    // https://malcoded.com/posts/angular-dynamic-components
+    const el = (cmpRef.hostView as EmbeddedViewRef<any>).rootNodes[0];
```

Comment:
Mind creating a private method for this now that the comment and functionality are duplicated in the file?
```
private getRootNode(componentRef: ComponentRef): any {
  // Technique for retrieving the component's root node taken from here:
  // https://malcoded.com/posts/angular-dynamic-components
  return (componentRef.hostView as EmbeddedViewRef<any>).rootNodes[0];
}
```

Reply:
Done